modules/varscan/1.0/config/default.yaml

```diff
@@ -0,0 +1,46 @@
+lcr-modules:  
+    varscan:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id}
+            sample_bam: "__UPDATE__"
```

Comment:
Shouldn't sample_bai virtually always also be an input even if not explicitly specified in the shell command?